tests/test_astromKPM.py

```diff
@@ -86,6 +86,8 @@ def test_af1(self):
         for band in ('i', 'r'):
             catalog, expected = self.load_data(('AF1', band))
             result = task.run(catalog, 'AF1_design')
+            print(result.measurement)
```

Comment:
Probably want to remove these print statements before merging.

Reply:
Thanks. Indeed I had meant to remove these before merging. Thanks for catching.